packages/core/admin/admin/src/content-manager/pages/ListView/index.js

```diff
@@ -166,6 +166,16 @@ function ListView({
     async (endPoint, source) => {
       getData();
 
+      // TODO: to remove, it is just a test to check the API
+      try {
+        const data = await fetchClient.get(
+          '/content-manager/collection-types/api::category.category/actions/countManyEntriesDraftRelations?ids=[1,2]'
+        );
```

Comment:
```suggestion
        const data = await fetchClient.get(
          `/content-manager/collection-types/api::category.category/actions/multipleEntriesNumberOfDraftRelations?${stringify(
            { ids: [1, 2] },
            { encode: false }
          )}`
        );
```

imo arrays should always be encoded, with this encoding you will not need to JSON parse the array in the backend controller side.

Reply:
I removed the test code you commented and I have understood how to call the API in the FE iteration of the ticket but if I try the API from POSTman and I remove the JSON parse I have problems, it always give me
<img width="1284" alt="Schermata 2023-06-30 alle 17 15 54" src="https://github.com/strapi/strapi/assets/2589748/76b33f77-3fee-417f-a8af-213c282136d7">
 back 0

Reply:
could you send the follwing query params? `ids[0]=1&ids[1]=2`

Reply:
yes in this way it works

Reply:
To make things consistent with the api I would use that format here. Wdyt?

Reply:
![Uploading Schermata 2023-06-30 alle 17.34.07.png…]()